flask_dynamo/manager.py

```diff
@@ -148,6 +149,40 @@ def _get_ctx(app):
                 'flask-dynamo extension not registered on flask app'
             )
 
+    @staticmethod
+    def _init_session(app):
+        session_kwargs = {}
+        # Only apply if manually specified: otherwise, we'll let boto
+        # figure it out (boto will sniff for ec2 instance profile
+        # credentials).
+        if app.config['AWS_ACCESS_KEY_ID']:
```

Comment:
That is a good question. In theory, ``self._init_settings()`` should be protecting the config, making this unnecessary. I'm not sure if we should lean on that here, or always do a safe ``app.config.get()``